JShellAPI/src/main/java/org/togetherjava/jshellapi/rest/ApiEndpoints.java

```diff
@@ -0,0 +1,16 @@
+package org.togetherjava.jshellapi.rest;
+
+/**
+ * This class holds endpoints mentioned in controllers. The main objective is to keep endpoints
+ * synchronized with testing classes.
```

Comment:
Don't use
> This class

And don't mention the tests, a class outside of tests shouldn't exist for the purpose of tests.